README.md

```diff
@@ -1367,6 +1369,71 @@ const info = await ant.getInfo();
 
 </details>
 
+### `getHandlers()`
+
+Retrieves the handlers supported on the ANT
+
+```typescript
+const handlers = await ant.getHandlers();
+```
+
+<details>
+  <summary>Output</summary>
+
+```json
+[
+  "evolve",
+  "_eval",
+  "_default",
+  "transfer",
+  "balance",
+  "balances",
+  "totalSupply",
+  "info",
+  "addController",
+  "removeController",
+  "controllers",
+  "setRecord",
+  "removeRecord",
+  "record",
+  "records",
+  "setName",
+  "setTicker",
+  "initializeState",
+  "state"
+]
+```
+
+</details>
+
+### `validate({ validations })`
+
+Validate the APIs exposed by the ANT. Defaults to verifying the requisite handlers are returned.
+
+```typescript
+async function validateSetTicker({ant}) {
+    /// ... your CUSTOM validation here
+    return true | false
+  };
+const validations = {
+  ["setTicker"] = validateSetTicker,
+};
+const validities = await ant.validate({validations});
+```
+
+<details>
+  <summary>Output</summary>
+
+```json
+{
+  "setTicker": { "valid": true },
+  "state": { "valid": false, "error": "...some error about why its not valid" }
+  /// rest of handler validations
+}
```

Comment:
just use a real example and show those outputs

Reply:
https://github.com/ar-io/ar-io-sdk/pull/237/commits/1529f795824a7b8352a47d2ee4f8a9048ba4fb91